core/pta/veraison_attestation/sub.mk

```diff
@@ -0,0 +1,11 @@
+srcs-$(CFG_VERAISON_ATTESTATION_PTA) += veraison_attestation.c
+srcs-$(CFG_VERAISON_ATTESTATION_PTA) += cbor.c
+srcs-$(CFG_VERAISON_ATTESTATION_PTA) += hash.c
+srcs-$(CFG_VERAISON_ATTESTATION_PTA) += sign.c
+
+cflags-y += -Wno-declaration-after-statement
```

Comment:
It's a pity we need to disable these two warnings because of the QCBOR header files.
It's out of the scope of this PR to fix that, but in the longer term, we should look into how to temporarily disable warnings in a header file.

Reply:
That should be `cflags-$(CFG_VERAISON_ATTESTATION_PTA) += ...` and same for the next line

Reply:
DONE